### PR TITLE
ci: prod sync

### PR DESCRIPTION
**Pull request by preethamr**
## Description
Prod sync

## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/agents/sequencer/src/lib/operations/execute/fastpath.ts [hunk 1]
```diff
@@ -119,14 +119,20 @@ export const storeFastPathData = async (bid: Bid, _requestContext: RequestContex
     await channel.assertExchange(config.messageQueue.exchanges[0].name, config.messageQueue.exchanges[0].type, {
       durable: config.messageQueue.exchanges[0].durable,
     });
+    const queue = config.messageQueue.queues.find((it) => it.name == transfer.xparams!.originDomain);
+    await channel.prefetch(queue?.limit || 1);
```
Comment (wanglonghong) [inline packages/agents/sequencer/src/lib/operations/execute/fastpath.ts:123]: I don't think this is needed before the `publish`

**Review by wanglonghong — CHANGES_REQUESTED** [review]

File: packages/agents/sequencer/src/lib/operations/execute/slowpath.ts [hunk 2]
```diff
@@ -72,6 +72,13 @@ export const storeSlowPathData = async (executorData: ExecutorData, _requestCont
     await channel.assertExchange(config.messageQueue.exchanges[0].name, config.messageQueue.exchanges[0].type, {
       durable: config.messageQueue.exchanges[0].durable,
     });
+    const queue = config.messageQueue.queues.find((it) => it.name == transfer.xparams!.originDomain);
+    await channel.prefetch(queue?.limit || 1);
```
Comment (wanglonghong) [inline packages/agents/sequencer/src/lib/operations/execute/slowpath.ts:76]: same here as above

**Review by wanglonghong — CHANGES_REQUESTED** [review]

File: packages/agents/sequencer/src/lib/operations/execute/slowpath.ts (hunk 2 above)
Comment (preethamr) [reply packages/agents/sequencer/src/lib/operations/execute/slowpath.ts]: You are right, removed it.

File: packages/agents/sequencer/src/lib/operations/execute/fastpath.ts (hunk 1 above)
Comment (preethamr) [reply packages/agents/sequencer/src/lib/operations/execute/fastpath.ts]: You are right, removed it.

**Review by wanglonghong — APPROVED** [review]

**Review by rhlsthrm — APPROVED** [review]

**Pull request merged**